pipeline/api/cloud.py

```diff
@@ -434,6 +435,17 @@ def upload_pipeline(
         """
         if new_pipeline_graph._has_run_startup:
             raise Exception("Cannot upload a pipeline that has already been run.")
+        if sys.version_info.major != 3 or (
+            sys.version_info.major == 3 and sys.version_info.minor != 9
+        ):
+            print(
+                "WARNING: pipeline-ai is still in development and the"
+                " upload_pipeline function has only been tested in Python 3.9. "
+                "We strongly recommend you use Python 3.9 as pipelines uploaded"
+                " in other Python versions are known to be broken. We are working"
+                "on adding support for 3.10 and 3.8; join us on Discord"
+                " https://discord.com/invite/7REbAX5v3N if you'd like to help us!"
```

Comment:
I don't think we can make this call-to-action as it likely requires work on the backend.

Reply:
The Mystic jobs page no longer exists, so Oscar and I figured send them to Discord and go from there (but with the aim that somebody seriously keen will speak to us and potentially be hired). Do you think I should clarify, i.e. "if you'd like to work with us"?

Reply:
I don't think this is appropriate place to try to hire people but leave that up to you and Oscar 😄